documentation/CLI_usage.md

```diff
@@ -0,0 +1,195 @@
+# CLI Usage
+
+---
+## npd
+
+### Description
+napari plugin devtools scans the current python environment to find out details about napari plugin information.
+
+### Usage
+```
+npd [COMMANDS] [ARGS...]
+```
+
+### Commands
+|Command        |Description                                          |
+|---------------|-----------------------------------------------------|
+|validate       |validate the given package is correctly setup        |
+|discover       |discover all plugins under current python environment|
+
+
+## npd validate
+
+### Description
+Validate package(s) within current python environment, report PASSED or FAILED for each check, 
+only return 1 when all checks passed.
+
+### Usage
+```
+npd validate [Options] <package> [<packages>...]
+```
+
+### Options
+|name, shorthand    |Description                     |
+|-------------------|--------------------------------|
+|--verbose, -v      |Show more detailed output       |
+
+### Examples
+
+validate single package
+```
+$ pip install napari-svg
+$ npd validate napari-svg
+----------------------------------------------------------------
+Scanning current python environment: /Library/Frameworks/Python.framework/Versions/3.8
```

Comment:
I prefer to scan my current directory or be able to specify a directory instead of my current python environment so that I can validate my plugin as I'm developing it and before I publish to Pypi. But if we are going to keep this method, I really like that you've added this output so that I know where the tool is scanning; it's very helpful.